yarn.lock

```diff
@@ -1908,12 +1908,12 @@
   resolved "https://registry.yarnpkg.com/@guardian/commercial-core/-/commercial-core-0.24.0.tgz#6ba387bd80bbed14561d073de5e7573b2fb5f663"
   integrity sha512-dBtJX+6mzjYJcy6hXMJ5w5oKDZu+Ob18z+vXZtkN+j3Bp2ex+jQik6IuQTwfn+KkN6VEMoIzYK1ZN3fqPHMwYw==
 
-"@guardian/consent-management-platform@6.11.5":
-  version "6.11.5"
-  resolved "https://registry.yarnpkg.com/@guardian/consent-management-platform/-/consent-management-platform-6.11.5.tgz#93681cbf61de0fadb4b65d835d7fc445cb847b31"
-  integrity sha512-ap7b09qk8I/EfoWB3NR8kT63hBpm4ATFGpRicCQ61DLMw2PgCWunMvizdbSmo5wNCQKPFgmX9cfRT5IGfPneKw==
+"@guardian/consent-management-platform@9.x.x":
+  version "9.0.1"
+  resolved "https://registry.yarnpkg.com/@guardian/consent-management-platform/-/consent-management-platform-9.0.1.tgz#bfcfca3eb32a65faa0f60ff723a294ba840d9953"
+  integrity sha512-ZN4tf7VpGxQu1CcqRsHNzWK61Q3KlaLGf8HKEX2oGotKzWZdoCx22tEA/v5Stdlv1ZkcMSqnfmtF1fJ5STgWGg==
   dependencies:
-    "@guardian/libs" "^1"
+    "@guardian/libs" "1.6.1 - 3.3.0"
```

Comment:
Maybe we want to allow all versions of @guardian/libs v3. For the other repo.

Reply:
Can't we let yarn work out what version it needs?